koch.nim

```diff
@@ -517,15 +513,6 @@ proc runCI(cmd: string) =
     when defined(posix):
       execFold("Run nimsuggest tests", "nim c -r nimsuggest/tester")
 
-    ## remaining actions
-    when defined(posix):
-      kochExecFold("Docs", "docs --git.commit:devel")
-      kochExecFold("C sources", "csource")
```

Comment:
Are we sure nightlies doesn't rely on this command?

Reply:
Nightlies don't use `koch runCI`. They run [nightlies.sh](https://github.com/nim-lang/nightlies/blob/master/nightlies.sh).